docs/services/slam/cartographer/_index.md

```diff
@@ -175,19 +189,71 @@ See [Attributes](#attributes) for details.
 }
 ```
 
-{{% /expand%}}
+{{% /tab %}}
+{{< /tabs >}}
+
+1. Start a mapping session
+
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+   On the cartographer panel, you can start a mapping session.
 
-#### Update existing map
+   When you start a mapping session:
 
-This mode is similar to [Create new map](#create-new-map), except it creates a new version of an existing map in the robot's location. (It does not overwrite the old version of the map.)
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+     Because Cartographer's algorithm is CPU-intensive when creating or updating a map, in this mode the `cartographer` module on your robot acts as a stub.
+   - Viam's [Data Capture](/services/data/configure-data-capture/) continuously monitors and captures your robot's sensor data while the robot is running.
+   - Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the map.
 
-The configuration is similar to the configuration for [creating a new map](#create-new-map), except you additionally configure the following **Attribute**:
+   Once you click **End session**, the map is uploaded to the cloud(?) and visible on your **Location** page under **SLAM library**.
```

Comment:
@EshaMaharishi if I understand correctly, the map is being built in the cloud, so I think it is odd to say it is the uploaded to the cloud. Would this be correct:

 ```suggestion
   Once you click **End session**, the map is saved and visible on your **Location** page under **SLAM library**.
```

Reply:
Good point, yes it is being built in the cloud. Hmm, how about `the map is saved to cloud storage`?